SH4RE/SH4RE/FirebaseDB.swift

```diff
@@ -76,18 +75,19 @@ func documentDelete(collectionPath : String, documentID : String, data:Dictionar
     return deleteSucess
 }
 
-func documentRead(collectionPath : String, documentID : String, completion: @escaping ((DocumentSnapshot)?)->Void){
+func documentRead(collectionPath : String, documentID : String, ret_doc: @escaping(Dictionary<String, String>)  -> Void) {
     let db = Firestore.firestore()
-    db.collection(collectionPath).document(documentID).addSnapshotListener { (docSnapshot, error) in
-        if let error = error {
-            print("Error reading document: \(error)")
-            completion(nil)
-        }else{
-            print("Document sucessfully read")
-            completion(docSnapshot)
+    let docRef = db.collection(collectionPath).document(documentID)
```

Comment:
are these brians changes, are these supposed to be overwriting?

Reply:
yeah i didnt touch this